table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -346,28 +340,29 @@ public UdbTable getTable(String tableName) throws IOException {
       List<UdbPartition> udbPartitions = new ArrayList<>();
       if (partitionColumns.isEmpty()) {
         PartitionInfo.Builder partitionInfoBuilder = PartitionInfo.newBuilder()
-            .setDbName(getUdbContext().getDbName())
+            .setDbName(mGlueDbName)
             .setTableName(tableName)
             .addAllDataCols(GlueUtils.toProto(table.getStorageDescriptor().getColumns()))
             .setStorage(GlueUtils.toProto(table.getStorageDescriptor(), pathTranslator))
             .setPartitionName(tableName)
-            .putAllParameters(table.getParameters());
+            .putAllParameters(tableParameters);
         udbPartitions.add(new GluePartition(
             new HiveLayout(partitionInfoBuilder.build(), Collections.emptyList())));
       } else {
         for (Partition partition : partitions) {
           String partName = GlueUtils.makePartitionName(partitionColumns, partition.getValues());
-          PartitionInfo.Builder pib = PartitionInfo.newBuilder()
-              .setDbName(getUdbContext().getDbName())
+          PartitionInfo.Builder partitionInfoBuilder = PartitionInfo.newBuilder()
+              .setDbName(mGlueDbName)
               .setTableName(tableName)
               .addAllDataCols(GlueUtils.toProto(partition.getStorageDescriptor().getColumns()))
               .setStorage(GlueUtils.toProto(partition.getStorageDescriptor(), pathTranslator))
               .setPartitionName(partName)
-              .putAllParameters(partition.getParameters());
+              .putAllParameters(partition.getParameters() == null
+                  ? Collections.emptyMap() : partition.getParameters());
           if (partition.getValues() != null) {
-            pib.addAllValues(partition.getValues());
+            partitionInfoBuilder.addAllValues(partition.getValues());
           }
-          udbPartitions.add(new GluePartition(new HiveLayout(pib.build(),
+          udbPartitions.add(new GluePartition(new HiveLayout(partitionInfoBuilder.build(),
               statsMap.getOrDefault(partName, Collections.emptyList()))));
```

Comment:
Is this the only use of `statsMap`? We may be able to just get rid of this variable and here, we could just use `Collections.emptyList()`.

Reply:
I have changed to `Collections.emptyList()`, and I notice the latest glue client supports the column statistics, added to TODO.